carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/core/capability/AbstractCapabilities.java

```diff
@@ -51,6 +51,10 @@
 
     protected DesiredCapabilities initBaseCapabilities(DesiredCapabilities capabilities, String browser, String testName) {
 
+        String platformName = Configuration.getPlatformName();
```

Comment:
we already have `Configuration.getPlatform()` which is organized correctly. let's reuse this method and remove your fresh in Configuration.java class